### ISSUE TITLE
Disable rather than silently removing packages when the maintainer changes

### ISSUE THREAD

**Issue by glenntanner3** [alexheretic/aurto#83]
So I didn't even know there was a maintainer check, it's good, but b/c it removes the packages rather than stepping over and flagging, I had no idea some of my packages where removed and not receiving their updates. Because of this, I have disabled the maintainer check. Unfortunately I understand there isn't a good way to make a warning during 'pacman -Syu', but removing without prompting the user I dont feel is the best way to do it. Perhaps another list, disabled_packages, could be created? Then when it's populated aurto commands could warn about disabled packages and status could tell what they are.

/etc/aurto/{maintainers,disabled_packages}

Disabled packages: enable by adding them again
google-chrome

**Comment by alexheretic** [comment]
Yes packages from untrusted maintainers are removed from the aurto repo. There'll be a warning in the log. As you say you can disable these checks.

After removal the repoless package will also turn up in `pacman -Qm` queries, which if you are using repos for all aur packages will usually be empty.

That's how i manage it, i have some check to ensure there are no repoless packages. Actually the same problem can come from the official repos E.g. they can remove a package that was installed and one of the only ways you can only know about it is running `pacman -Qm`. So this strategy covers both.

Perhaps there are other ways we could also notify the user. A log of removed packages is one, but in a way we have that already as it is possible to scan the journal logs. I also considered having opt in desktop notifications, but I'm unsure if that'll actually be useful for most users.

I could also add the existing strategies to the readme or wiki. Would that have been helpful?

**Comment by glenntanner3** [comment]
I hadn't thought about Qm, thank you. However I think maintaining a simple list would be beneficial. If this process isn't in the readme/wiki it should be and probably in the short section of the how to use/install section. However I doubt I would have taken the time to read it outside the installation/usage section.

```
# list disabled during status
if [ -s /etc/aurto/disabled_packages ];then
echo "Disabled package: add again to enable"
cat /etc/aurto/disabled_packages
fi

# remove package
echo "${package}" >> /etc/aurto/disabled_packages

# remove during add package
sed "/${package}/d" /etc/aurto/disabled_packages
```
